container/lxd/server.go

```diff
@@ -131,6 +131,16 @@ func (s *Server) CreateClientCertificate(cert *Certificate) error {
 	return errors.Trace(s.CreateCertificate(req))
 }
 
+// GetServerEnvironmentCertificate returns the current server environment
+// certificate
+func (s *Server) GetServerEnvironmentCertificate() (string, error) {
```

Comment:
Instead of a new method, I would just populate a new member for the certificate in the call to ```NewServer```. We already call ```GetServer``` there.